components/centraldashboard/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Step 1: Builds and tests
-FROM node:10-alpine AS build
+FROM node:10.13-alpine AS build
```

Comment:
https://hub.docker.com/_/node
node:10-alpine is node 10.22

Why do we need to revert to node:10.13?